Classes/Domain/Model/Document.php

```diff
@@ -637,8 +637,6 @@ public function addCollection(Collection $collection): void
      * Removes a collection
      *
      * @param Collection $collection
-     *
-     * @return ObjectStorage<Collection> collections
      */
     public function removeCollection(Collection $collection)
```

Comment:
This should have a `void` return value. You removed the (wrong) return value from PHPDoc, but didn't add it to the function's signature. Or am I missing something?
```suggestion
    public function removeCollection(Collection $collection): void
```